python/ql/lib/semmle/python/security/dataflow/CleartextLoggingCustomizations.qll

```diff
@@ -41,7 +41,8 @@ module CleartextLogging {
    */
   class SensitiveDataSourceAsSource extends Source, SensitiveDataSource {
     SensitiveDataSourceAsSource() {
-      not SensitiveDataSource.super.getClassification() = SensitiveDataClassification::id()
+      not SensitiveDataSource.super.getClassification() =
```

Comment:
NIT: we usually prefer `in [...]` for readability, although using `= [...]` is the same :shrug: 

```suggestion
      not SensitiveDataSource.super.getClassification() in
```